### DISCUSSION TITLE
SAE schaltet Wärmepumpe leider nicht ein.

### DISCUSSION THREAD

**Discussion by DrMabuse100** [camueller/SmartApplianceEnabler#448] [Installation und Betrieb]
Hallo, ich habe so wie in der Anleitung 2 Geräte für das WP. Setup angelegt. Aber leiter wir die nicht eingeschaltet.
Anbei das Setup.

![SHM einst](https://user-images.githubusercontent.com/111784088/215251098-130af4f3-8083-4606-8479-cc4314503a44.JPG)

![WP Zähler](https://user-images.githubusercontent.com/111784088/215251121-a96ba474-24ec-466e-9c82-83fce5ef8a4d.JPG)
![WP](https://user-images.githubusercontent.com/111784088/215251129-a9a3144b-efb1-4d49-b36b-85c24ae2b946.JPG)
![WP+Pv Schaltzeiten](https://user-images.githubusercontent.com/111784088/215251136-1f364e31-265d-4a32-9d28-89199784e441.JPG)
![WP-PV schalter](https://user-images.githubusercontent.com/111784088/215251139-c9e7e87d-9130-4fa6-8107-cb5e3a68dcd9.JPG)
![Wp-PV](https://user-images.githubusercontent.com/111784088/215251141-3002722a-7667-40e4-9ee8-b191ded520ee.JPG)

Was habe ich da evtl. noch falsch gemacht ?

Dankeschön Freundlicher Gruß

**Comment by camueller** [comment]
Hast Du 3 kW Überschuss über längere Zeit? Bei der aktuellen Witterung würde ich ds bezweifeln ...

**Comment by DrMabuse100** [comment]
Hallo danke für den Hinweis,

aber ich dachte der SHM macht das und überstuert dann die vorgabe vom SAE (siehe Screnshot) oder mache ich einen denkfehler ?
Wie meist du das genau mit längerzeit genau ?  

Dankeschön Freundlicher Gruß

**Comment by camueller** [comment]
Bei der Einschaltoption geht es darum, die SG-Ready-Funktion von Wärmepumpen zu unterstützen, um zukünftig Energiebedarfe vorzuziehen und dafür Überschussenergie zu nutzen.

Alles weitere steht hier: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/SwitchOnOption_DE.md
